pagopa/io-app
# feat: [IOPID-1501] Adapt LandingScreen to the new DS

## Short description
This PR addresses the redesign of the LandingScreen by adopting the new DS. 
It's been used as reference to write this [DS Getting Started Document](https://pagopa.atlassian.net/wiki/spaces/IAEI/pages/944734806/DS+getting+started).

> [!TIP]
> Review Privacy ✅
>
> [![Run e2e tests](https://github.com/pagopa/io-app/actions/workflows/test-e2e.yml/badge.svg?branch=IOPID-1501-landing-screen)](https://github.com/pagopa/io-app/actions/workflows/test-e2e.yml)

<details><summary>🎦 Demo 🎦</summary>
<p>

| Android a11y | iOS a11y | Session Expired | Session Expired LV |
| - | - | - | - | 
| <video src="https://github.com/pagopa/io-app/assets/16268789/40853c09-3967-48e7-96bd-c6efe3007ad8" /> | <video src="https://github.com/pagopa/io-app/assets/16268789/61943639-59b8-4e36-a386-d69ece0aa1b8" /> | <video src="https://github.com/pagopa/io-app/assets/16268789/8e4826e8-5991-4d25-994f-c3965ef6c022" /> | <video src="https://github.com/pagopa/io-app/assets/16268789/3dd2b041-1e48-4cea-b6e8-06fdabb22f98" /> 

| Small Device Carousel & Legacy Jailbroken Device Modal | Android Tablet and Legacy EIC not supported modal presentation | Legacy iOS EIC Not Supported modal presentation |
| - | - | - | 
| <video src="https://github.com/pagopa/io-app/assets/16268789/6cfb9af4-05c5-4a59-b213-a31ff9f2010c" /> | <video src="https://github.com/pagopa/io-app/assets/16268789/2ce3a03d-278e-43ee-a959-38841eda66d8" /> | <video src="https://github.com/pagopa/io-app/assets/16268789/d6f4988f-0c8a-4073-b0db-141d37e12c9e" /> |

| EIC UAT |
| - |
| <video src="https://github.com/pagopa/io-app/assets/16268789/cc688a12-2e54-46cc-9041-1a621c6d3852" /> |

</p>
</details> 

## How to test
- Run on a device and test both FL and Standard Login session expiration.
- Test a11y on both Android and iOS devices.
- Run the app on a jailbroken device and test the complementary informative modal.
- Test  Android and iOS CIE not supported informative modal presentation.
- Test the informative compatibility alert on Android Tablet devices.
- Test the CIE UAT enable/disable functionality by tapping three times on the carousel dots.
- Check UI also with the DS Flag active.
- Test that the last link in the page it correctly navigates to the Privacy Policy.
